src/utils/assets.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright 2023 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License atCONST_GOOGLE_LOGO
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const CONST_GOOGLE_LOGO =
```

Comment:
nits: Add JSDoc for the format, and the name does not need to include CONST.

Reply:
Done.